feature-libs/cart/_index.scss

```diff
@@ -20,3 +19,9 @@ $useLatestStyles: true;
 @import './quick-order/index';
 @import './wish-list/index';
 @import './import-export/index';
+
+@import 'cxbase/blocks/modal';
+
+@import 'misc/hamburger';
```

Comment:
Took a glance, but I don't think the cart styles is using anything from the hamburger stylesheet right?